js/Picker.web.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @flow
+ * @format
+ */
+
+import React from 'react';
+import type {
+  ViewStyleProp,
+  TextStyleProp,
+} from 'react-native/Libraries/StyleSheet/StyleSheet';
+import type {ViewProps} from 'react-native/Libraries/Components/View/ViewPropTypes';
```

Comment:
We should try to avoid importing react-native on web because it increases the bundle size by a lot. I think the latest RNWeb gets rid of a lot of these props, so we should be able to remove these in the future.